flink-ml-iteration/flink-ml-iteration-common/src/main/java/org/apache/flink/iteration/operator/HeadOperator.java

```diff
@@ -498,6 +534,11 @@ public HeadOperatorStatus getStatus() {
         return status;
     }
 
+    @VisibleForTesting
+    public SpillableFeedbackChannel getFeedbackChannel() {
```

Comment:
The feedback channel is used by both `HeadOperator` and `TailOperator`, so we cannot close it in `HeadOperator` in case of `TailOperator` is still using it.